utils/e2e-tests/ts/tests/rpc/eth.ts

```diff
@@ -28,6 +28,15 @@ describe("eth rpc", () => {
     expect(network.chainId).toBe(5234n);
   });
 
+  it("has the expected initial dev balances", async () => {
+    const [alice, bob] = devSigners;
+    const aliceBalance = await provider.getBalance(alice);
+    const bobBalance = await provider.getBalance(bob);
+
+    expect(Number(aliceBalance)).toBe(eth.INITIAL_DEV_ACCOUNT_BALANCE);
```

Comment:
Creating a new account and transferring fund to it would be preferrable - this would not break when/if we change the initial dev account balance